File: week1:C/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do{
        n = get_int("Height: ");
    }
    while (n>8 || n<1);
    for (int i=0;i<n;i++){
        for (int j=n-i-1;j>0;j--){
            printf(" ");
        }
        for (int z=0;z<i+1;z++){
            printf("#");
        }
        printf("\n");
    }
}

File: week6:Python/sentimental-mario-more/mario.py
# TODO

from cs50 import get_int

while True:
    n=get_int("Height:")
    if n<=8 and n>=1:
        break

for i in range(n):
    for j in range(n-i-1):
        print(" ",end="")
    for z in range(i+1):
        print("#",end="")
    print("  ",end="")
    for z in range(i+1):
        print("#",end="")
    print("")